code/modules/fish/fish_items.dm

```diff
@@ -152,15 +152,15 @@
 	name = "shark teeth"
 	desc = "A number of teeth, supposedly from a shark."
 	icon = 'icons/obj/fish_items.dmi'
-	icon_state = "teeth"
```

Comment:
The comment of Contrabang still stands here. Mapping tools use the `icon_state` which is set on the definition level of an object